src/Components/GameSession.js

```diff
@@ -6,17 +6,19 @@ import wordImporter from '../funcLib/WordImporter.js';
 import PlayAgainButton from './PlayAgainButton.js';
 import { Col, Container, Row } from 'react-bootstrap';
 import checkForBingo from '../funcLib/CheckForBingo';
-import { useOutletContext } from 'react-router-dom';
+import { useOutletContext, useParams } from 'react-router-dom';
```

Comment:
If multiple params, does the code still work as expected?

Reply:
Yes!

Reply:
Multiple params are included like: 

[localhost:3000/play/[params1]/[params2] ](localhost:3000/play/params1/params2)

then referenced in order when useParams() is called